src/sagas.js

```diff
@@ -13,11 +13,17 @@ function fetchData(schema, since, until) {
       is_weekly: true,
       schema
     },
-    url: 'https://staging.api.bounties.network/analytics/'
+    url: 'http://localhost:8000/analytics'
```

Comment:
we need to pass this through as an env variable :)

Reply:
this is my bad, I was using the local API because it had support for the categories
will change it back to https://api.bounties.network/analytics/